helpers.json

```diff
@@ -61,6 +61,10 @@
       "name": "h_canActivateAcidChozo",
       "requires": ["SpaceJump"]
     },
+    {
+      "name": "h_ShaktoolVanillaFlag",
+      "requires": []
```

Comment:
Add a note saying this is the shaktool room unmodified behaviour when entering from the right.